packages/devtools_app/lib/src/screens/memory/panes/diff/controller/simple_controllers.dart

```diff
@@ -0,0 +1,10 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class RetainingPathController {
+  final hideStandard = ValueNotifier<bool>(true);
```

Comment:
Nit: I generally try and avoid exposing `ValueNotifier`s directly. If these are just meant to be toggled, maybe use `ValueListenable` getters and add `toggleHideStandard()` and `toggleInvert()` methods?